src/Pool/Pool.router.ts

```diff
@@ -31,13 +31,7 @@ export class PoolRouter extends Router {
     /**
      * Get pool
      */
-    this.router.get(
-      '/projects/:id/pool',
-      withAuthentication,
-      withProjectExists,
```

Comment:
Is this one not necessary anymore? `withProjectExists`
I'm not sure if it's a requirement for a pool to have a valid (active) project, we could ask

Reply:
on the frontend side, isn't a requirement.
 @cazala @belohlavek ?